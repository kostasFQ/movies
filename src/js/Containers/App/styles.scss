@import '../../../../node_modules/normalize.css/normalize.css';

body {
  min-width: 450px;
}

#APP {
  display: flex;
  flex-direction: column;
}

*:focus {
  outline: 0;
}
