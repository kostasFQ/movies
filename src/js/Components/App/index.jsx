import { hot } from 'react-hot-loader';
import React from 'react';
import './styles.scss';
import Header from '../Header';

const App = () => (
  <div>
      <Header/>
  </div>
);

export default hot(module)(App);
