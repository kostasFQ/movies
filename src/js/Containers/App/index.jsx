import { hot } from 'react-hot-loader';
import React from 'react';
import './styles.scss';
import Content from '../Content';

const App = () => (
  <div>
    <Content />
  </div>
);

export default hot(module)(App);
