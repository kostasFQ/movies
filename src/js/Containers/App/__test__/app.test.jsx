import React from 'react';
import renderer from 'react-test-renderer';
import App from '../index';
import { Provider } from 'react-redux';
import reduser from '../../../redusers';
import { createStore } from 'redux';

const store = createStore(reduser);
test('test component render', () => {
  const render = renderer.create(
    <Provider store={store}>
      <App />
    </Provider>,
  );
  const component = render.toJSON();

  expect(component).toMatchSnapshot();
});
