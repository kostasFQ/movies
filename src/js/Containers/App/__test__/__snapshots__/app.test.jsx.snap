// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component render 1`] = `
<div>
  <div
    className="container"
  >
    <h1
      className="title"
    >
      FILMS
    </h1>
    <div>
      <input
        className="input"
        placeholder="search"
        type="text"
      />
    </div>
  </div>
  <div>
    <img
      alt="pic"
      className="poster"
      src="test-file-stub"
    />
    <div
      className="container"
    >
      <div
        className="div"
      >
        <div
          className="container"
        >
          <h1
            className="h1"
          >
            THE JUNGLE BOOK
          </h1>
          <ul
            className="list"
          >
            <li
              className="item"
            >
              Adventure
            </li>
            <li
              className="item"
            >
              Drama
            </li>
            <li
              className="item"
            >
              Family
            </li>
            <li
              className="item"
            >
              Fantasy
            </li>
            <span>
              1h 46m
            </span>
          </ul>
          <div
            className="container"
          >
            <img
              alt="star"
              className="star"
              height="20px"
              src="test-file-stub"
              width="20px"
            />
            <img
              alt="star"
              className="star"
              height="20px"
              src="test-file-stub"
              width="20px"
            />
            <img
              alt="star"
              className="star"
              height="20px"
              src="test-file-stub"
              width="20px"
            />
            <img
              alt="star"
              className="star"
              height="20px"
              src="test-file-stub"
              width="20px"
            />
            <img
              alt="star"
              className="star"
              height="20px"
              src="test-file-stub"
              width="20px"
            />
            <div
              className="ratingBox"
            >
              4.8
            </div>
          </div>
        </div>
        <div
          className="container"
        >
          <div
            className="buttonsBlock"
          >
            <button
              className="container"
              onClick={null}
              type="button"
            >
              Watch Now
            </button>
            <button
              className="container"
              onClick={[Function]}
              type="button"
            >
              View Info
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
